content/docs/rfcs/template/README.md

```diff
@@ -0,0 +1,62 @@
+---
+slug: XX
+title: XX/(WAKU2|LOGOS|CODEX|...)-TEMPLATE
```

Comment:
Assuming RFCs that does not belong to a single project or applies to several (e.g. "RLN" without relay) would just not use a prefix at all?

Reply:
Sounds good to me! Don't think we need too overexplain this - a "*" would fully indicate the meaning, I think.

Reply:
ff47efb

Reply:
I like that :).
What I did in the template was to be backwards compatible. But... I'd have to update the RFCs adding the `category` anyway. Ill put that into the restructure issue (yet to be opened, see #513)

Reply:
Hm, I'm worried this complicates things? The optional prefix seems like most minimal KISS.

Do we have an example of some spec project using double namespacing? E.g. <ID>/<PROJECT>/<SPEC>.

What do we hope to gain from this, vs more informal prefix?